Makefile.am

```diff
@@ -212,6 +212,7 @@ libarchive_la_SOURCES= \
 	libarchive/archive_write_set_format_7zip.c \
 	libarchive/archive_write_set_format_ar.c \
 	libarchive/archive_write_set_format_by_name.c \
+    libarchive/archive_write_set_format_filter_by_ext.c \	
```

Comment:
Please use the same indentation as the rest of this file.

Please alphabetize these lists (this should go down 2 lines).